src/kirby/components/segmented-control/segmented-control.component.tns.html

```diff
@@ -0,0 +1,15 @@
+  <StackLayout orientation="horizontal" horizontalAlignment="left" class="segment-control">
+    <ng-container *ngFor="let item of segmentItems">
+      <GridLayout columns="auto, auto" (tap)="onSegmentTap(item)">
+        <ng-container *ngIf="item.badge">
+          <FlexBoxLayout class="segment-badge-wrapper" col="1" flexDirection="row" alignItems="center" justifyContent="flex-end">
```

Comment:
I honestly don't know what we prefer, flexbox layout rules in CSS or view-templates.... for web, it's a given that it goes in CSS... not sure here? (I actually think it's quite readable, and {N} view-templates can't be considered a candidate for "semantic web")...

I need somebody else to decide / voice their opinion on this.